join.html

```diff
@@ -61,30 +61,29 @@ <h4>Fee Structure</h4>
     <table class="table">
       <tbody>
         <tr>
-          <td><strong>Backer Hacker</strong></td>
+          <td><strong>Standard Hacker</strong></td>
           <td>
             <ul>
-              <li>$100 per month, no further discounts.</li>
+              <li>$100 per month. Rates can be reduced to $60 per month by becoming a 'Volunteer Hacker.'</li>
             </ul>
           </td>
         </tr>
         <tr>
-          <td><strong>Standard Hacker</strong></td>
+          <td><strong>Volunteer Hacker</strong></td>
           <td>
             <ul>
-              <li>$60 per month, or</li>
-              <li>$55 per month when paying 3 months or more in advance (up to 12 months)</li>
+              <li>$60 per month. To be on this rate you must complete 4 hours of volunteer work for HSBNE per month. Don't be shy! There's always plenty to do.</li>
             </ul>
           </td>
         </tr>
         <tr>
-          <td><strong>Starving Hacker</strong></td>
+          <td><strong>Starving Volunteer Hacker</strong></td>
           <td>
             <ul>
               <li>$30 per month, no further discounts</li>
-              <p>To be eligible for this level you must be genuinely unable to afford the standard tier. Being a
-                student or unemployed generally makes you eligible. You will require proof of eligibility during your
-                induction.</p>
+			  <li>To be on this rate you must complete 4 hours of volunteer work for HSBNE per month.</li>
```

Comment:
drop the 4 hours thing, just like above, by signing a volunteer agreement and meeting it each month. maybe some words about higher time expectations than standard volunteer. Maybe point out the default volunteering position is to attend the monthly working bee